content/modules/redisai/redisai-quickstart.md

```diff
@@ -0,0 +1,16 @@
+---
+Title: RedisAI Quick Start Tutorial
+description:
+weight: 20
+alwaysopen: false
+categories: ["Modules"]
+---
+For this quick start tutorial, we recommend that you use:
+
+- Either:
+    - [A Redis Enterprise Software database with the RedisAI module enabled]({{< relref "/modules/create-database-rs.md" >}})
+    - [A Redis Cloud Pro database with the RedisAI module enabled](https://redislabs.com/redis-enterprise-cloud/)
```

Comment:
we can't deliver this out of the box for now

Reply:
@bmansheim I would rather have them reach out to us if they want this.  I meant that redisai is not zero-touch available from our website to create a database.  there is no set of instructions, it's more a manual operation our side

Reply:
@K-Jo So I’ll take it out. Do we want to say something like “Contact Support if you want to try out RedisAI on Cloud”?

Reply:
Yes please add this